### ISSUE TITLE
PeterBot should squash commits automatically

### ISSUE THREAD

**Issue by MattIPv4** [cdnjs/cdnjs#13390]
## Idea

Currently, once a pull request has been approved by the CI, it may still have multiple commits associated with it which we like to have squashed down into a singular commit.

Additionally, if the commit history for a PR becomes too long, @PeterBot will simply raise an error asking the user to reduce the commit history length.

In both situations, it would be awesome if @PeterBot could automatically squash all existing commits in the PR into a single commit, so that further changes can be made to be tested against the CI or before the final review from maintainers is requested.

 - Automatically squash commits once a PR is approved by CI
 - Automatically squash commits if commit history on a PR becomes too long

## Relevant Scripts

CI Script: https://github.com/cdnjs/cdnjs-drone-ci/blob/master/cdnjs.sh
PR Review Script: https://github.com/PeterDaveHello/Drone-GitHub-status-report-bot/blob/master/main.sh

**Comment by MattIPv4** [comment]
This is no longer needed as PeterBot is no longer in operation.
